spacy/sklearn.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+import spacy
+
+
+__all__ = [
+    'SpacyTransformer',
+    'DOC', 'VECTOR', 'SENTENCES', 'ENTITIES', 'NOUN_CHUNKS',
+    'SENTIMENT', 'TOKENS'
+    ]
+
+
+DOC = 'self'
+VECTOR = 'vector'
+SENTENCES = 'sents'
+ENTITIES = 'ents'
+NOUN_CHUNKS = 'noun_chunks'
+SENTIMENT = 'sentiment'
+TOKENS = 'tokens'
+
+
+class SpacyTransformer(BaseEstimator, TransformerMixin):
+
+    def __init__(self,
+                 model=spacy.en.English(),
```

Comment:
I agree that requiring the model to be provided would be easiest, but sklearn have some pretty explicit guidelines about how their estimators should be init'd

>The arguments accepted by __init__ should all be keyword arguments with a default value. In other words, a user should be able to instantiate an estimator without passing any arguments to it.

additionally,

>There should be no logic, not even input validation, and the parameters should not be changed. The corresponding logic should be put where the parameters are used, typically in `fit`

(from http://scikit-learn.org/stable/developers/contributing.html#instantiation)

The rationale behind these guidelines is to ensure that the estimator would be fully compatible with their model selection utilities, which make extensive use of `get_params` and `set_params`